aim/db/models.py

```diff
@@ -83,6 +90,20 @@ def to_attr(self, session):
         return res_attr
 
 
+class NetflowVMMExporterPol(model_base.Base, model_base.HasDisplayName,
+                            model_base.HasAimId, model_base.AttributeMixin,
+                            model_base.IsMonitored, model_base.HasName):
+    """DB model for Netflow VMM Exporter"""
+    __tablename__ = 'aim_netflow_exporter_pol'
+    __table_args__ = (model_base.uniq_column(__tablename__, 'name') +
+                      model_base.to_tuple(model_base.Base.__table_args__))
+
+    dst_addr = sa.Column(sa.String(16))
```

Comment:
is 16 bytes enough for the dest_addr and src_addr? Do we have to consider IPV6 address here?

Reply:
Its safe to support ipv6 as well. Made changes to support it in the latest commit.